### DISCUSSION TITLE
suggestion: keep a CHANGELOG.md or similar

### DISCUSSION THREAD

**Discussion by ghost** [vincjo/datatables#106] [Ideas]
When maintaining a mid/big size project, keeping track of dependencies gets tricky.

I'm using version 1.14.5, and I see the current version is 1.14.10. From the version number alone it seems no major/breaking changes have been made. However I see big commits (like [this one](../commit/53d0d97710135897cac1f56c00b7fa649419d181)) where it seems a lot has changed.

I can of course go over all the changes myself and check if any api's are different, but I am not familiar with this project's internals so I might miss something. I think it would be nice if there was some sort of changelog file, or a description of each release on Github releases or something. At [changelog.md](https://changelog.md) they explain it better :)

Something that users of this project can quickly reference to see if there are breaking changes or if we can safely update.

awesome library btw, keep up the good work!!

**Comment by vincjo** [comment]
Thanks for your suggestion. I will definitely add a changelog to document developments over time.
